### DISCUSSION TITLE
Non-blocking! Class named Text in this project

### DISCUSSION THREAD

**Discussion by ivofernandes** [singerdmx/flutter-quill#693] [General]
My issue is about [Mobile]

I have tried running `example` directory successfully before creating an issue here. 

Using the version 4.0.5 of this package I run into the error:

Error: 'Text' is imported from both 'package:flutter/src/widgets/text.dart' and 'package:flutter_quill/src/models/documents/nodes/leaf.dart'.
          Text('Font size', style: Theme.of(context).textTheme.headline6),
          ^^^^

The problem probably is the naming of this class that is the same of the Text Widget of flutter, probably there is an easy workaround as is something with imports, but 

https://github.com/singerdmx/flutter-quill/blob/master/lib/src/models/documents/nodes/leaf.dart

To solve the problem we can always use the material Text in another file/widget to avoid the problem in the imports

**Comment by mark8044** [comment]
Agree, this is a bit weird.  For now I imported material twice, 

```
import 'package:flutter/material.dart';
import 'package:flutter/material.dart' as mat;
```
and then use `mat.Text` when I need to use the Text widget.  Probably best for flutterQuill to move away from using the name `Text` though, you are right

**Comment by kairan77** [comment]
how about 

import 'package:flutter_quill/flutter_quill.dart' hide Text;
